requirements.txt

```diff
@@ -1,27 +1,27 @@
-astroalign~=2.5.1
-astropy~=5.3.4
+astroalign~=2.6.0
+astropy~=6.1
 astroquery~=0.4.7
 barycorrpy~=0.4.4
-colour_demosaicing~=0.2.5
+colour_demosaicing~=0.2.6
 dynesty~=1.2.3;platform_system=='Windows'
 holoviews~=1.19.1
 importlib-metadata>=3.6;python_version<='3.7'
 imreg_dft~=2.0.0
 LDTk~=1.8.4
 lmfit~=1.3.2
-matplotlib~=3.7.5
-numpy~=1.24.4
-pandas~=1.5.3
-panel~=1.4.5
-photutils~=1.13.0
+matplotlib~=3.9.2
+numpy~=1.26.4
```

Comment:
Something worth keeping an eye out on the future is updating to bumpy 2.0 once our other dependents do so.

Reply:
Most of them are Numpy 2.x compatible now. But we couldn't update Numpy due to an inconsistency with LDTk. So the Numpy version was rolled back to the most recent that has not caused an issue. Numpy 1.26.4 is the last in the 1.x line and released in February 2024. Until LDTk is updated (or we can perhaps update Numba and fix another package hdbscan==0.8.39) any Numpy update will be inconsistent. It will also require deleting some cached HDB files to reset them. 
``` 
ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject```

Bottom line is it's the best we can do for now, I think.